src/main/scripts/org/reaktivity/specification/amqp/link/transfer.to.client.at.least.once/server.rpt

```diff
@@ -0,0 +1,129 @@
+      [0x01]                                                      # number of fields in list = 1
+      [0xa1 0x09 0x6c 0x6f 0x63 0x61 0x6c 0x68 0x6f 0x73 0x74]    # container-id = "localhost"
+
+# begin frame exchange
+read [0x00 0x00 0x00 0x1b]                                        # length = 27
+     [0x02]                                                       # dof = 2
+     [0x00]                                                       # type = AMQP (0)
+     [0x00 0x01]                                                  # channel = 1
+     [0x00 0x53 0x11]                                             # performative = begin (17)
+     [0xc0 0x0e]                                                  # list type (list8), physical length = 14
+     [0x04]                                                       # number of fields in list = 4
+     [0x40]                                                       # remote-channel = null
+     [0x52 0x01]                                                  # next-outgoing-id = 1
+     [0x70 0x00 0x00 0x07 0xff]                                   # incoming-window = 2047
+     [0x70 0x7f 0xff 0xff 0xff]                                   # outgoing-window = 2147483647
+
+write [0x00 0x00 0x00 0x1d]                                       # length = 29
+      [0x02]                                                      # dof = 2
+      [0x00]                                                      # type = AMQP (0)
+      [0x00 0x01]                                                 # channel = 1
+      [0x00 0x53 0x11]                                            # performative = begin (17)
+      [0xc0 0x10]                                                 # list type (list8), physical length = 16
+      [0x04]                                                      # number of fields in list = 4
+      [0x60 0x00 0x01]                                            # remote-channel = 1
+      [0x52 0x01]                                                 # next-outgoing-id = 1
+      [0x70 0x00 0x00 0x3f 0xff]                                  # incoming-window = 16383
+      [0x70 0x7f 0xff 0xff 0xff]                                  # outgoing-window = 2147483647
+
+# attach frame exchange
+read [0x00 0x00 0x00 0x17]                                        # length = 23
+     [0x02]                                                       # dof = 2
+     [0x00]                                                       # type = AMQP (0)
+     [0x00 0x01]                                                  # channel = 1
+     [0x00 0x53 0x12]                                             # performative = attach (18)
+     [0xc0 0x0a]                                                  # list type (list8), physical length = 10
+     [0x03]                                                       # number of fields in list = 3
+     [0xa1 0x05 0x6c 0x69 0x6e 0x6b 0x31]                         # name = "link1"
```

Comment:
For string or symbol fields, recommend using literal string in script for readability.
```
[0xa1 0x05] "link1"                         # name = "link1"
```